lib/Rules.js

```diff
@@ -23,8 +24,37 @@ Rules.prototype.run = function(data) {
       throw new Error("rules must have an `if` and `then` object");
     }
 
-    if (this.testsMatch(rule["if"], data)) {
-      this.runOutcomes(rule.then, outcomeResults);
+    if (_.isPlainObject(rule["if"])) {
+
+      if (this.testsMatch(rule["if"], data)) {
+        this.runOutcomes(rule.then, outcomeResults);
+      }
+      else {
+        if (rule.elseThen) {
+          this.runOutcomes(rule.elseThen, outcomeResults);
+        }
+      }
+    }
+    else if (_.isArray(rule["if"])) { // can track N number of rule sets, i.e. series of OR conditions
+
+      var trackFailedValidation = [];
+
+      for (var j = 0; j < rule["if"].length; j++) {
+        var item = rule["if"][j];
+        if (!this.testsMatch(item, data)) {
+          trackFailedValidation.push(true);
```

Comment:
What would be more appropriate is to see if one of them passes, and then break. So we could use `testPassed = true;` and then `break`. This is because the array acts as a series of `OR` blocks, so if at least one is true, the `then` section will run. If we `break` on one failure, we won't know if the other ones pass.